include/deal.II/base/vectorization.h

```diff
@@ -782,7 +782,7 @@ class VectorizedArray<double>
    * Actual data field. Since this class represents a POD data type, it
    * remains public.
    */
-  __m512d data;
+  __m512d data{0.};
```

Comment:
Is there any reason to no just do 
```suggestion
  __m512d data = _mm512_set1_pd(0.);
```
instead of defining a conversion constructor and an explicit default constructor, and similarly below?